<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFundsToSavingDialog</class>
 <widget class="QDialog" name="AddFundsToSavingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddFundsToSavingDialog</string>
  </property>
  <widget class="QComboBox" name="financePicker">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select the finance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addFundsPushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add funds</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueLineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="currencyISOLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="savingTitle">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddFundsToSavingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>612</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
